packages/checkout/sdk/src/types/smartCheckout.ts

```diff
@@ -252,7 +252,7 @@ export type ERC20ItemRequirement = {
   /** The type to indicate this is a ERC20 item requirement. */
   type: ItemType.ERC20;
   /** The amount of the item. */
-  contractAddress: string;
+  tokenAddress: string;
   /** The contract address of the ERC20. */
```

Comment:
I noticed the type doc comments for amount/token address are the wrong way around, could you update them in this PR and also change the comment to say token address instead of contract

Reply:
Nice catch! updated.